File: Finals/loops/for7-3.cpp
#include<iostream>
using namespace std;

int main()
{
    int n=7;
    for(int i=0;i<n;i++)
    {
        for(int spaces=0;spaces<i;spaces++)
        cout<<" ";
        cout<<"*";
        for(int spaces=0;spaces<2*(n-i-1);spaces++)
        cout<<" ";
        if(i!=n-1)
        cout<<"*";
        cout<<"\n";
    }
}

File: Finals/loops/for7-3.py
n=7
for i in range(7):
    for spaces in range(i):
        print(" ",end="")
    print("*",end="")
    for spaces in range(2*(n-i-1)):
        print(" ",end="")
    if(i!=n-1):
        print("*",end="")
    print("")